pytest_girder/pytest_girder/fixtures.py

```diff
@@ -21,9 +21,9 @@ def bcrypt():
     """
     Mock out bcrypt password hashing to avoid unnecessary testing bottlenecks.
     """
-    with mock.patch('bcrypt.hashpw') as hashpw:
-        hashpw.side_effect = lambda x, y: x
-        yield hashpw
+    from passlib.hash import bcrypt
+    with mock.patch('girder.models.user.bcrypt', new=bcrypt.using(rounds=4)) as fastBcrypt:
```

Comment:
Is 4 rounds very fast?

Reply:
It's the lowest we can go: https://passlib.readthedocs.io/en/stable/lib/passlib.hash.bcrypt.html#passlib.hash.bcrypt

Reply:
Is there reason to run the hashing inside the test at all rather than make it a no-op?

Reply:
We should probably have one test that does run it, but the rest should mock it with a no-op IMO.